index.html

```diff
@@ -0,0 +1,161 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Capiston-project-1</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <div class="container">
+      <header class="header">
+        <nav>
+          <div class="navs">
+            <div class="top-nav">
+              <a href="#"><img class="top-item" src="assets/facebook.png" alt="facebook"></a>
+         <a href="#"><img class="top-item" src="assets/twitter.png" alt="twitter"></a>
+               <a href="#" class="top-item">English</a>
+                <a href="#" class="top-item">My Page</a>
+                <a href="#"class="top-item">Logout</a>
+            </div>
+            <div class="bottom-nav">
+               <a href="about.html"> <img class="nav-logo" src="assets/logo.jpg" alt="logo"></a>
+            <div class="nav-links">
+                <a class="nav-link" href="about.html">About</a>
+                <a class="nav-link" href="#partner">Partner</a>
+                <a class="nav-link" href="#">Join</a>
+                <a class="nav-link" href="#">Sponsor</a>
+                <a class="nav-link" id="kbu" href="#">CC-CAMPAIN</a>   
+            </div>
```

Comment:
[OPTIONAL] Good job so far in implementing the required changes,  It will also be nice to make your text alignment conform to the given design, Matching your design to the given design ensures a cohesive visual identity across different components, screens, or materials. Consistency helps create a unified experience for users and maintains brand recognition and professionalism. :slightly_smiling_face: 

Below is an example of the difference: :arrow_down: 

<table>
    <tr>
      <td>Your Design</td>
      <td>Figma Design</td>
    </tr>
    <tr>
      <td>

![StudentNav](https://github.com/fatima-najafi/capstone-project-module1/assets/43528594/4c421c32-4bb8-4e9d-b90c-ff7fc8730c80)

</td>
      <td>

![Design Alignment](https://github.com/fatima-najafi/capstone-project-module1/assets/43528594/49d02143-91b3-402f-a04d-6f6194e1cece)

</td>
    </tr>
  </table>